pydfs_lineup_optimizer/lineup_exporter.py

```diff
@@ -12,9 +12,10 @@ def __init__(self, lineups):
     @staticmethod
     def render_player(player):
```

Comment:
You can pass custom render function to export, so if you need another format it's better to write another function and pass it to export. `exporter.export('result.csv', render_player_function)` I'm not sure about changing current default function.